packages/blade/src/components/Button/IconButton/IconButton.stories.tsx

```diff
@@ -14,6 +14,7 @@ const Page = (): ReactElement => {
     <StoryPageWrapper
       componentName="IconButton"
       componentDescription="Useful for making clickable icons. For example - close button for modals, inputs, etc."
+      apiDecisionLink=""
```

Comment:
Why is this empty? Let's make it optional instead?

Reply:
when we skip this prop, apiDecisionLink is auto-generated from componentName prop since our URL format is mostly constant.

This prop is to override that link. So you can override it with new link, or override it with blank string to remove that section itself.

We can't treat undefined here because undefined would just create a API decision link. Maybe I can change this to null instead.

Reply:
Maybe this will be more meaningful.
```suggestion
      apiDecisionLink="none"
```

Reply:
done. Added `apiDecisionLink={null}` instead